coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/transformers/FromUnixtimeOperatorTransformer.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino.transformers;
+
+import java.util.List;
+
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlLiteral;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+import com.linkedin.coral.common.functions.FunctionReturnTypes;
+import com.linkedin.coral.common.transformers.SqlCallTransformer;
+
+import static org.apache.calcite.sql.type.ReturnTypes.*;
+import static org.apache.calcite.sql.type.SqlTypeName.*;
+
+
+/**
+ * This transformer operates on SqlCalls with 'FROM_UNIXTIME(x)' Coral IR function
+ * and transforms it to trino engine compatible function - FORMAT_DATETIME(FROM_UNIXTIME(x)).
+ * For Example:
+ * A SqlCall of the form: "FROM_UNIXTIME(10000)" is transformed to
+ * "FORMAT_DATETIME(FROM_UNIXTIME(10000), 'yyyy-MM-dd HH:mm:ss')"
+ */
+public class FromUnixtimeOperatorTransformer extends SqlCallTransformer {
+
+  private static final String FORMAT_DATETIME = "format_datetime";
+  private static final String FROM_UNIXTIME = "from_unixtime";
+
+  @Override
+  protected boolean condition(SqlCall sqlCall) {
+    return sqlCall.getOperator().getName().equalsIgnoreCase(FROM_UNIXTIME);
```

Comment:
@findinpath If we make this check not simply based on the name so we make sure it only matches the Hive operator and not the Trino one, would we still need `TimestampFromUnixtimeTransformer` from #464? Also, can the operator in this case live only in `coral-trino`?

Reply:
`from_unixtime` takes for both Trino and Hive a numeric value, I don't know how to distinguish it
- Trino reference https://trino.io/docs/current/functions/datetime.html?highlight=from_unixtime#from_unixtime returning timestamp(3) with time zone#
- Hive reference https://cwiki.apache.org/confluence/display/hive/languagemanual+udf returning string type

Reply:
Discussed offline. We can use `timestamp_from_unixtime` as a different name to disambiguate. @findinpath will try it.

Reply:
https://github.com/linkedin/coral/pull/467